app/helpers/application_helper.rb

```diff
@@ -47,12 +47,12 @@ def projects_namespace?
 
   def last_commit(project)
     if project.repo_exists?
-      time_ago_in_words(project.commit.committed_date) + " ago"
+      time_ago_in_words(project.commit.committed_date) + t("global.ago")
```

Comment:
Due to word ordering, it should be written as:

`_("%s ago") % [time_ago_in_words(project.commit.committed_date)]`

(sorry, I dont familiar with ruby, but familiar with gettet technologies)